shoes-core/lib/shoes/text.rb

```diff
@@ -17,6 +17,10 @@ def app
       parent && parent.app
     end
 
+    def links
+      Shoes::Link.find_links(@texts)
+    end
```

Comment:
not 100% sure about using a singleton method to find the links... e.g. that is publicly accessible but doesn't make much sense outside of text/text_block. I'd have probably gone with a module to include... that said, we could also put it into its own singleton or sort of helper object `LinkFinder`... not leaning strong in any way - any ideas?